<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SafheAsli</class>
 <widget class="QMainWindow" name="SafheAsli">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(108, 184, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>200</width>
      <height>170</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">

QListWidget {
    border-radius: 10px;
    background-color: rgb(255, 255, 200);
	font: 700 12pt &quot;Segoe Print&quot;;
	border-color: rgb(30, 14, 255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>0</y>
      <width>581</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTextBrowser" name="textBrowser_asli">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>29</y>
        <width>500</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QTextBrowser {
    border-radius: 10px;
    background-image: url(:/source/back_chat.png);
	
	border-color: rgb(30, 14, 255);
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>515</y>
        <width>50</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 10px;
    padding: 10px;
    background-color: #376fa6;
	color: &quot;white&quot;;
	
}

QPushButton:hover {
    background-color: #007600;
}

QPushButton:pressed {
    background-color: #888888;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_asli">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>515</y>
        <width>450</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border-radius: 10px;
    padding: 5px;
	background-color: rgb(255, 255, 255);

}
</string>
      </property>
     </widget>
     <widget class="QLabel" name="name_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>501</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    border-radius: 6px;
	
	font: 700 12pt &quot;Segoe Print&quot;;

	background-color: rgb(255, 85, 127);

	border-color: rgb(85, 85, 0);

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QListWidget" name="group_list">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>220</y>
      <width>200</width>
      <height>170</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QListWidget {
    border-radius: 10px;
    background-color: rgb(190, 255, 150);
	font: 700 12pt &quot;Segoe Print&quot;;
	border-color: rgb(30, 14, 255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="channel_list">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>420</y>
      <width>200</width>
      <height>170</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QListWidget {
    border-radius: 10px;
    background-color: rgb(255, 220, 150);
	font: 700 12pt &quot;Segoe Print&quot;;
	border-color: rgb(30, 14, 255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>41</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Chats</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Groups</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>400</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Channels</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuoptions">
    <property name="title">
     <string>opt</string>
    </property>
    <addaction name="actionLog_out"/>
   </widget>
   <widget class="QMenu" name="menuadd">
    <property name="title">
     <string>add</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionnew_chat"/>
    <addaction name="actionnew_group"/>
    <addaction name="actionnew_channel"/>
    <addaction name="actionjoin_group"/>
    <addaction name="actionjoin_channel"/>
   </widget>
   <addaction name="menuoptions"/>
   <addaction name="menuadd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLog_out">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/source/ham_icon.png</normaloff>:/source/ham_icon.png</iconset>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
  <action name="actionnew_chat">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/source/newchat.png</normaloff>:/source/newchat.png</iconset>
   </property>
   <property name="text">
    <string>new chat</string>
   </property>
  </action>
  <action name="actionnew_group">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/source/newgroup.png</normaloff>:/source/newgroup.png</iconset>
   </property>
   <property name="text">
    <string>new group</string>
   </property>
  </action>
  <action name="actionnew_channel">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/source/newchannel.png</normaloff>:/source/newchannel.png</iconset>
   </property>
   <property name="text">
    <string>new channel</string>
   </property>
  </action>
  <action name="actionjoin_group">
   <property name="text">
    <string>join group</string>
   </property>
  </action>
  <action name="actionjoin_channel">
   <property name="text">
    <string>join channel</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
